src/components/button/button.js

```diff
@@ -72,7 +72,7 @@ module.exports = class Button extends Control {
     render() {
         return provide({
             block: this.bem.block,
-            attrs: {
+            attrs: Object.assign({
                 onMouseEnter: this._onMouseEnter.bind(this),
```

Comment:
maybe I missed it but is this the handlers like _onMouseEnter came from the base class? So why should we write them here?